### DISCUSSION TITLE
对于单词本导入功能的一些小的suggestion

### DISCUSSION THREAD

**Discussion by Zdp11one** [YiiGuxing/TranslationPlugin#4315] [Ideas]
作者大大你好，我今天尝试着将本地的单词文件导入到单词本中发现，如果一个单词有多个释义的话，在表中是一行显示的，但是在idea的单词本展示界面会自动分为多行
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/97154005/fa1e9e52-cc5a-444a-add4-0ab982252a14)

![image](https://github.com/YiiGuxing/TranslationPlugin/assets/97154005/3e24feb6-c99d-4235-acb0-529d6b7b0c8f)

这在使用中对于有轻微强迫症的我来说有点难受，您看这个是否可以优化一下？

**Comment by YiiGuxing** [comment]
单词本的导入并不会检测当前单词本内是否已有相同的单词存在，因此会出现单词重复的问题，重复的单词只能通过手动删除。所以单词本的导入一般只适用于从一个单词本上导入到一个空的单词本上的场景。

**提示：如果是想要单词本跨设备共享的话，建议通过指定单词本的存在路径到网盘同步文件夹内来实现。**
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/10737066/3ef600ce-787e-4849-8dc1-009e1912bf6e)
